### PR TITLE
Update README links to new repository name 'ECC'

### PR DESCRIPTION
**Pull request by karnavpargi**
Changed `everything-claude-code` to `ECC`

## What Changed
remaned paths to `/ECC` from `/everything-claude-code``

## Why This Change
Seen `/everything-claude-code` on number of places so updated it to `/ECC`

## Testing Done

- [ ] Manual testing completed
- [ ] Automated tests pass locally (`node tests/run-all.js`)
- [ ] Edge cases considered and tested

## Type of Change
- [ ] `fix:` Bug fix
- [ ] `feat:` New feature
- [ ] `refactor:` Code refactoring
- [X] `docs:` Documentation
- [ ] `test:` Tests
- [ ] `chore:` Maintenance/tooling
- [ ] `ci:` CI/CD changes

## Security & Quality Checklist
- [ ] No secrets or API keys committed (ghp_, sk-, AKIA, xoxb, xoxp patterns checked)
- [ ] JSON files validate cleanly
- [ ] Shell scripts pass shellcheck (if applicable)
- [ ] Pre-commit hooks pass locally (if configured)
- [ ] No sensitive data exposed in logs or output
- [ ] Follows conventional commits format

## Documentation
- [] Updated relevant documentation
- [ ] Added comments for complex logic
- [X] README updated (if needed)

---
## Summary by cubic
Renamed all README links and references from `affaan-m/everything-claude-code` to `affaan-m/ECC`, and updated examples to use the `ECC` checkout or relative paths.
Updates badges, community links, issues/PRs and releases, install commands, marketplace commands, repo identifiers, troubleshooting/PR references, and the Star History chart.

<sup>Written for commit 5ecee70196770c30aa4f2cb9511a7a2e38ff2b84. Summary will update on new commits. <a href="https://cubic.dev/pr/affaan-m/ECC/pull/1986?utm_source=github">Review in cubic</a></sup>

## Summary by CodeRabbit

* **Documentation**
  * Updated README to reference the canonical affaan-m/ECC repository across badges, community links, changelog/releases, marketplace install guidance, naming/migration notes, clone/copy/manual install instructions, FAQ links, PR references, repository settings, and Star History chart source.

[Review Change Stack](https://app.coderabbit.ai/change-stack/affaan-m/ECC/pull/1986?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ This is intentional. Anthropic marketplace/plugin installs are keyed by a canoni
 
 ```bash
 # Clone the repo first
-git clone https://github.com/affaan-m/everything-claude-code.git
+git clone https://github.com/affaan-m/ECC.git
 cd everything-claude-code
```
Comment (karnavpargi) [inline README.md]:
```suggestion
cd ECC
```
